apps/avatax/e2e/tests/product_without_tax_class.spec.ts

```diff
@@ -0,0 +1,245 @@
+import { it, describe } from "vitest";
+import { e2e } from "pactum";
+import { string } from "pactum-matchers";
+import { gql } from "../utils";
+
+describe("Product without assigned tax class", () => {
+  const testCase = e2e("Product without tax class");
```

Comment:
Can we use the same tests name as in the testmo?
"App should calculate taxes for checkout with product without tax class [pricesEnteredWithTax: False]"

Reply:
And please add link to testmo